init/eessi_environment_variables

```diff
@@ -106,6 +106,12 @@ if [ -d $EESSI_PREFIX ]; then
             false
           fi
 
+        # Fix wrong path for RHEL >=8 libcurl 
+	rhel_libcurl_file="/etc/pki/tls/certs/ca-bundle.crt"
```

Comment:
```suggestion
	# This is required here because we ship curl in our compat layer. If we only provided
	# curl as a module file we could instead do this via a `modluafooter` in an EasyBuild
	# hook (or via an Lmod hook)
	rhel_libcurl_file="/etc/pki/tls/certs/ca-bundle.crt"
```